### PR TITLE
Add fields for virtual meeting links and phone meeting dial-in

### PR DESCRIPTION
**Pull request by pjaudiomv**
ref https://github.com/bmlt-enabled/bmlt-root-server/issues/281

### CODE REVIEW

**Comment by pjaudiomv** [comment]
changing field names, i think the ones make more sense jbone suggested here https://github.com/bmlt-enabled/bmlt-root-server/issues/281#issuecomment-602255962

**Comment by jbraswell** [comment]
I think this should be `virtual_meeting_link` and `phone_meeting_link`.

The CREATE TABLE statements also have to change.

And then there need to be database migrations to add the fields to existing root servers.

**Comment by pjaudiomv** [comment]
why does the CREATE TABLE statement need to change

File: main_server/local_server/install_wizard/sql_files/InitialMeetingsData.sql [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-(0, 'location_neighborhood', 'Neighborhood', 'en', NULL, 'Neighborhood', NULL, NULL),
-(0, 'location_municipality', 'Town', 'en', NULL, 'Town', NULL, NULL),
-(0, 'location_sub_province', 'County', 'en', NULL, 'County', NULL, NULL),
-(0, 'location_province', 'State', 'en', NULL, 'State', NULL, NULL),
-(0, 'location_postal_code_1', 'Zip Code', 'en', NULL, NULL, 0, NULL),
-(0, 'location_nation', 'Nation', 'en', NULL, 'Nation', NULL, NULL),
-(0, 'comments', 'Comments', 'en', NULL, 'Comments', NULL, NULL),
-(0, 'train_lines', 'Train Lines', 'en', NULL, NULL, NULL, NULL),
-(0, 'bus_lines', 'Bus Lines', 'en', NULL, NULL, NULL, NULL),
+(0, 'meeting_name', 'Meeting Name', 'en', 0, 'Meeting Name', NULL, NULL),
```
Comment (jbraswell) [inline main_server/local_server/install_wizard/sql_files/InitialMeetingsData.sql]: Being explicit about `visibility` being `0` seemed better, want to make sure a fresh install still works right.

**Comment by pjaudiomv** [comment]
fields arent showing up on new install for me, they are in database but not in admin ui

**Pull request merged**